WindowsMRAssetConverter/FileSystem.cpp

```diff
@@ -4,6 +4,19 @@
 #include "stdafx.h"
 #include "FileSystem.h"
 
+std::wstring FileSystem::GetRelativePathWithTrailingSeparator(std::wstring from, std::wstring to)
+{
+    wchar_t outPath[MAX_PATH] = L"";
+    if (PathRelativePathTo(outPath, from.c_str(), FILE_ATTRIBUTE_DIRECTORY, to.c_str(), FILE_ATTRIBUTE_DIRECTORY))

```

Comment:
This API will only work on desktop. Can you please use std::filesystem instead?

Reply:
Ok, np. I'll take that as a go-ahead to replace the other two shlwapi calls (PathFindFileName, PathRemoveExtension) that were already in the code.

Reply:
A closer inspection reveals that msvc doesn't support std::filesystem::relative (c++17), which would be the natural replacement for PathRelativePathTo. I was really hoping to avoid re-implementing that.